### DISCUSSION TITLE
Nue 1.0 beta Docs

### DISCUSSION THREAD

**Discussion by seneca** [nuejs/nue#286] [General]
Thank you Tero for letting us know about the latest beta release candidate. What's the best way to report typos, etc (that's all I can do at the moment). What's the branch name that I should be doing this in?

**Comment by tipiirai** [comment]
Thanks! Here is the upcoming website and the documentation:

https://www2.nuejs.org/

And here is the source code: 

https://github.com/nuejs/nue/tree/dev/packages/nuejs.org

I think the easiest way to contribute is to hit the edit button from the markdown docs

<img width="372" alt="image" src="https://github.com/user-attachments/assets/c332a972-67f8-4757-a2bc-0f894da89fbb">

And hit save. I guess it makes a pull request from there, or do you need more access rights?

Thank you for the help!

**Comment by seneca** [comment]
Thanks Teri,
This should be good. Will let you know if need any access rights added.
Bes regards
Les

PS: this is shaping up to be a great release. :-)

**Comment by seneca** [comment]
Hi Tero,
Ups, no I can’t edit these files in place. I need to fork them and propose changes.
You could give me access rights for editing in place but I understand that you many not want to so I will try to fork the repo first.
Best regards
Les

**Comment by seneca** [comment]
Hi Tero,
I’ve just read https://www2.nuejs.org/blog/nue-1-beta/ article and would like to suggest a few changes in terms of a very few mistakes like double words, etc.
Where in the repo can I do that if I can be of any help?
Les

**Reply by nobkd** [reply]
This should always contain the latest changes: https://github.com/nuejs/nue/blob/dev/packages%2Fnuejs.org%2Fblog%2Fnue-1-beta%2Findex.md
